packages/spor-alert-react-native/src/ClosableAlert.tsx

```diff
@@ -38,3 +59,11 @@ export const ClosableAlert = ({
     </BaseAlert>
   );
 };
+
+const texts = {
+  close: {
+    nb: "Lukk",
+    sv: "Dölj",
```

Comment:
Dette er riktig ord, sant @stenkula?

Reply:
Jepp man bruker dölj som man skal skjule noe. For eksempel skjule et telefonnummmer = dölja et telefonnummer

Litt usikker på om man bruker dölj i formindelse med lukk da 😅
Da kan man sikkert også bruker stäng eller avbryt.

Reply:
Forhørte meg men mamma her. Hun mener dölj er riktig ord å bruke i denne settingen! 🙌

Reply:
![svensk kultur](https://media.giphy.com/media/QsNZWfg8pDMUOVCxPs/giphy.gif)